### PR TITLE
Feat: Tokeniser

### PR DESCRIPTION
**Pull request by miningape**
Added token class, as well as a scanner which can take the source text and turn it into a list of tokens.

(Forgot to make this a draft pr)

### CODE REVIEW

File: Token/Token.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from Token.TokenTypeEnum import TokenType
+from typing import Any
```
[the remark this answers is not in the document]
Comment (miningape) [reply Token/Token.py:2]: How do you mean? afaik you cannot enforce typing in python, and signatures are the typings given to fx. functions.
I'm personally enforcing it through vscode (strict mode) and pylint.

**Comment by miningape** [comment]
Depends on #9 (We can still get rid of `.pylintrc` if we dont want it)

**Comment by miningape** [comment]
Depends on #10 for REPL interface

**Review by spralja — APPROVED** [review]

**Comment by miningape** [comment]
@spralja why change the name of the directory containing all the code about tokens to pychart? Thats just confusing lmao

**Comment by spralja** [comment]
The problem with using libs or src directories, Is that importing the module requires some magic, whereas if there is just a pychart directory you can just import pychart from the parent directory.

**Comment by spralja** [comment]
@miningape Also at least there should exists a pychart module inside libs or src.

**Comment by miningape** [comment]
Yeah I see what you mean, maybe we just make inside `libs/pychart/` the files that we need externally, something like `runner.py`? 

Then in backend we can do this:
```python
from pychart import pychart_eval_line

pychart_eval_line('print("name jeff")')
```

**Comment by spralja** [comment]
This is confusing. When I merged tokenizer into spralja-package, why did it also merge the other way [around](https://github.com/miningape/pychart/pull/28)?

**Comment by spralja** [comment]
@miningape I feel like I made a billion useless commits, but now this works with package-2

**Comment by miningape** [comment]
@spralja can you stop committing without actually affecting the content of the tokeniser, this can have been solved later when we merge your package pr, right now we're just splitting the versions apart

**Comment by spralja** [comment]
@miningape ye I was just tesing, and merge tokenizer into a another branch but for some reason it also merge the other way aorund. [PR](https://github.com/miningape/pychart/pull/28)

**Pull request merged**